scripts/helpers/poll_quick_ci.py

```diff
@@ -0,0 +1,40 @@
+import argparse
+import subprocess
+import time
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("pr", help="Pull request number")
+    parser.add_argument("check", help="Pull request check to poll for")
+    args = parser.parse_args()
+
+    print(f"Gathering info on the {args.check} check being run for the current pull request.")
+    polling = True
+    poll_count = 0
+    poll_max = 300
+    while (polling):
+        for line in subprocess.run(f"gh pr checks -R project-chip/connectedhomeip {args.pr}", stdout=subprocess.PIPE, shell=True).stdout.decode("utf-8").splitlines():
+            if args.check in line:
```

Comment:
This feels fragile - you're parsing the log line, but if the log line changes and you get back something that doesn't have args.check, or "pending", "pass" or "fail" then you'll loop forever. 

If you're going to use polling here, the count increment needs to be outside of the nested if.

Reply:
Moved increment outside.